index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name=”viewport” content=”width=device-width, initial-scale=1″ charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins:400,500,700,900' rel='stylesheet'>
+  </head>
+  <body>
+
+    <div id="container">
+
+      <div id="app-bar-compact">
+        <div id="toolbar">
+
+          <div id="div-button">
+            <nav id="menu">
+              <button id="icon-button-button" type="button" onclick="hamburgClick()">
+                <span id="union">
+                  <span class="vector"></span>
+                  <span class="vector"></span>
+                  <span class="vector"></span>
+                </span>
+              </button>
+            </nav>
+          </div>
+
+          <div id="div-logo"><a id="div-logo-a" href="#">My Logo</a></div>
+
+          <nav id="menu-2">
+            <button id="menu-2-button" type="button">Portfolio</button>
+            <button id="menu-2-button" type="button">About</button>
+            <button id="menu-2-button" type="button">Contact</button>
+          </nav>
+
+        </div>
+      </div>
+      <!-- end of container
+      <div id="header-shapes-mobile"></div>-->
+      <div id="headline">
```

Comment:
:red_circle: This container should be a section tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. Kindly update it